starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function(){
 
+}
     //Checking account deposit function
-
       //On click of the depositChecking button
-
+$("#depositChecking").click(function(event){
         //Get value from the amountChecking input field
-
```

Comment:
Since this code should be inside the document.ready function, it should be indented so it's easy to see where things open and close. Same with everything below this line as well.